UnitsNet/QuantityInfo.cs

```diff
@@ -123,6 +123,25 @@ public QuantityInfo(QuantityType quantityType, [NotNull] UnitInfo[] unitInfos, [
         ///     The <see cref="BaseDimensions" /> for a quantity.
         /// </summary>
         public BaseDimensions BaseDimensions { get; }
+
+        /// <summary>
+        /// Gets the <see cref="UnitInfo"/> which has <see cref="BaseUnits"/> that are a subset of the given <see cref="BaseUnits"/>.
+        /// </summary>
+        /// <param name="baseUnits">The <see cref="BaseUnits"/> to check against.</param>
+        /// <returns>The UnitInfo that has BaseUnits that are a subset.</returns>
+        public UnitInfo GetUnitInfoForBaseUnitsSubset(BaseUnits baseUnits)
+        {
+            if(baseUnits == null)
+                throw new ArgumentNullException(nameof(baseUnits));
+
+            var matchingUnitInfos = UnitInfos.Where((unitInfo) => unitInfo.BaseUnits.IsSubsetOf(baseUnits));
+            if(!matchingUnitInfos.Any())
+                throw new NotImplementedException("No units were found that are a subset of the given BaseUnits.");
+            else if(matchingUnitInfos.Skip(1).Any())
```

Comment:
`Skip(1).Any()` tickles me a bit :-) 

This is really minor, but I couldn't help myself trying out an alternative. How about something like this? It should be similarly optimized to enumerate as little as necessary.
```c#
           var matchingUnitInfos = UnitInfos
                .Where((unitInfo) => unitInfo.BaseUnits.IsSubsetOf(baseUnits))
                .Take(2)
                .ToArray();

            var firstUnitInfo = matchingUnitInfos.FirstOrDefault();
            if (firstUnitInfo == null)
                throw new NotImplementedException("No units were found that are a subset of the given BaseUnits.");
            
            if (matchingUnitInfos.Length > 1)
                throw new InvalidOperationException("More than one unit was found that is a subset of the given BaseUnits.");

            return firstUnitInfo;
```

Reply:
Sounds good to me. Fixed!